_usage/mvu.md

```diff
@@ -0,0 +1,123 @@
+                                       -B ${BABELFISH_HOME_NEW}/bin \
+                                       -d ${BABELFISH_DATA_OLD} \
+                                       -D ${BABELFISH_DATA_NEW} \
+                                       -p 5432 -P 5433 -j $JOBS --verbose
+```
+
+Refer to the [pg_upgrade documentation](https://www.postgresql.org/docs/current/pgupgrade.html) for detailed information about using `pg_upgrade`. 
+
+In the event of a failure, the new `data` directory (`$BABELFISH_DATA_NEW`) might have issues, but the old `data` directory (`$BABELFISH_DATA_OLD`) will remain intact. To retry the upgrade, you can remove the new `data` directory  and re-initialize it again. Follow the instructions in `Initiating the Data directory` from `INSTALLING.md`.
+
+After the upgrade is successfully completed, start the new server with `pg_ctl`:
+
+`${BABELFISH_HOME_NEW}/bin/pg_ctl -D ${BABELFISH_DATA_NEW} -l logfile start`
+
+To update the Babelfish extensions, connect to the PostgreSQL endpoint of the Babelfish database and execute the following statements from a single session in this order:
+
+```sql
+ALTER EXTENSION babelfishpg_common UPDATE;
+ALTER EXTENSION babelfishpg_tsql UPDATE;
+ALTER EXTENSION babelfishpg_tds UPDATE;
+```
+
+Some Babelfish specific GUCs/settings are considered critical and their values must be the same before and after a major version upgrade. Changing any of the parameter values during the upgrade might cause inconsistent database behavior or even data corruption. Therefore, it is mandatory that the following Babelfish related parameters are set to the same values after upgrade as they were before upgrade.
+
+- `babelfishpg_tds.tds_default_numeric_precision`
+- `babelfishpg_tds.tds_default_numeric_scale`
+- `babelfishpg_tsql.database_name`
+- `babelfishpg_tsql.default_locale`
+- `babelfishpg_tsql.migration_mode`
+- `babelfishpg_tsql.server_collation_name`
+
+You can modify parameter settings at the `psql` command line; connect with the command: 
+
+`$PKGDIR_NEW/bin/psql -U your_user_name -d babelfish_db_name` 
+
+Then run following SQL command to change above settings:
+
+`ALTER SYSTEM SET <setting name> = <setting value>;`
+
+Then, use the following command to reload the configuration file:
+
+`SELECT pg_reload_conf();`
+
+*Note:* If you change any of the following parameters, the server needs to be restarted:
+
+- `babelfishpg_tds.tds_default_numeric_precision`
+- `babelfishpg_tds.tds_default_numeric_scale`
+- `babelfishpg_tsql.server_collation_name`
+
+Run the following command to restart the server:
+
+`${BABELFISH_HOME_NEW}/bin/pg_ctl -D ${BABELFISH_DATA_NEW} -l logfile restart`
+
+Please note that you cannot change migration mode during an upgrade.  For example, if `babelfishpg_tsql.migration_mode` was `multi-db` before the upgrade, and you change it to `single-db`, then you will not able to see any user-defined databases that were there before the upgrade.
```

Comment:
This is more like an example of a scenario when user misses to set back the setting to the original value. So we should present this as an example and highlight it so that it doesn't get slipped though the reader's eyes.

Reply:
I've moved it up to where users are setting param values (after the upgrade)... please check the updated version.